src/index.html

```diff
@@ -3,6 +3,7 @@
 <head>
     <meta charset="utf-8">
     <title>Jellyfin</title>
+    <link rel="shortcut icon" href="favicon.ico">
```

Comment:
Failed attempt at getting webpack to include it, but I figured it should stay in